desktop/core/src/desktop/middleware_test.py

```diff
@@ -156,3 +156,21 @@ def test_ensure_safe_redirect_middleware():
     settings.MIDDLEWARE.pop()
     for finish in done:
       finish()
+
+def test_spnego_middleware():
+  done = []
+  # settings.MIDDLEWARE.append('desktop.middleware.SpnegoMiddleware')
+  try:
+    done.append(desktop.conf.AUTH.BACKEND.set_for_testing(['desktop.auth.backend.SpnegoDjangoBackend']))
+
+    c = make_logged_in_client()
+    response = c.get("/desktop/debug/is_alive")
+    assert_equal(200, response.status_code)
+
+    response = c.get("/")
+    assert_equal(401, response.status_code)
```

Comment:
Are those error code specific with Spengo backend? (as is_alive is not protected)

i.e. to properly test should we also look at the headers of the response for some properties set by the Spengo backend?